docs/dev_guide/ceph_s3_tests_guide.md

```diff
@@ -0,0 +1,143 @@
+# Ceph S3 Tests Guide
+
+## General Settings For Ceph Tests
+
+### 1) Before Building The Images (Noobaa-Core)
+We will use minikube to run the tests. It is recommended to build all images on the minikube docker daemon.  
+To configure your docker client to use minikube's docker run the following before running `make`:
+```bash
+eval $(minikube docker-env)
+```
+
+Note: For users with OS Mac with chip M1: you need to remove `mongoDB` installation in the test docker file `Tests.Dockerfile`, since there are no ARM packages available for it, in any case it is not used in this context.
```

Comment:
maybe wait with this remark as in my PR I'm removing this part completely

Reply:
I can remove it. Would you please add a link to the change?

Reply:
not sure I know how :) 
but as part of PR #7218 (Commit https://github.com/noobaa/noobaa-core/pull/7218/commits/e6bdc7682c833a390231d5330a1e2de364b2a758)

Reply:
I'll remove it, just waiting for more comments.